.ci/scripts/update-integration-testdata.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
I'd say add an entry in the CODEOWNERS for this file, otherwise, `robots-ci` will be the codeowners and we don't own it.

Reply:
Sure, added an entry with @elastic/elastic-agent-control-plane as owners (please, let me know if another team should own this). https://github.com/elastic/elastic-agent/pull/10620/commits/4a53ad59c69e6c2feb9acd4348be1805732e15fa

Also, I changed the backport to active-9 for now. Otel components dependencies are only being updated in 9.1 and 9.2, which makes the alignment with the latest kube-stack versions [harder](https://github.com/elastic/elastic-agent/pull/10350#issuecomment-3376372127).